tests/Dflydev/FigCookies/CookiesTest.php

```diff
@@ -19,13 +20,13 @@ class CookiesTest extends TestCase
      * @test
      * @dataProvider provideCookieStringAndExpectedCookiesData
      */
-    public function it_creates_from_request(string $cookieString, array $expectedCookies) : void
+    public function it_creates_from_request(string $cookieString, array $expectedCookies): void
     {
-        /** @var RequestInterface|ObjectProphecy $request */
-        $request = $this->prophesize(static::INTERFACE_PSR_HTTP_MESSAGE_REQUEST);
-        $request->getHeaderLine(Cookies::COOKIE_HEADER)->willReturn($cookieString);
+        /** @var RequestInterface|MockObject $request */
+        $request = $this->createMock(self::INTERFACE_PSR_HTTP_MESSAGE_REQUEST);
```

Comment:
prophecy is still usable, by loading a trait

Reply:
i have no idea if @simensen wants to keep prophecy

Reply:
I don't care about Prophecy. I think this was maybe added by @Ocramius at some point?